vendor-bin/propel2/composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "autoload-dev": {
+        "classmap": [
+            "../../tests/Bridge/Propel2/Model"
```

Comment:
you shouldn't need that as this should be inherited thanks to `theofidry/composer-inheritance-plugin`

Reply:
ok makes sense then